apps/docs/src/components/PageTemplate.astro

```diff
@@ -1,16 +1,36 @@
 ---
 import { parseComponent } from "@selleo/core";
 import { Code } from "astro/components";
-const { component: Component, name, ...props } = Astro.props;
+import { MobileViewIFrame } from "./MobileViewIFrame";
+const {
+  component: Component,
+  name,
+  mobileViewPath,
```

Comment:
`mobilePath` may be sufficient, wdyt?

Reply:
https://github.com/Selleo/design-system/pull/66#discussion_r1124311884 Same